KsApi/Service+RequestHelpers.swift

```diff
@@ -63,10 +65,13 @@ extension Service {
           let decodedObject = try JSONDecoder().decode(GraphResponse<A>.self, from: data)
           if let errors = decodedObject.errors, let error = errors.first {
             observer.send(error: .decodeError(error))
+            print("🔴 [KsApi] Failure decodeError \(error.message)")
```

Comment:
and separate hardcoded / dynamic values like `🔴 [KsApi] Failure - Decoding error: \(error.message)`?